WireUI/Sources/WireUIBase/Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,33 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+"tabBar.contacts.title" = "Contacts";
```

Comment:
The string has been moved and changed. Can I prevent somehow that the translations aren't lost?

Reply:
I personally don't know but assume crowdin has a translation memory and would expect that adding the new translations is very fast. Maybe you could forward a list of changed keys to whoever manager crowding? 🤔

Reply:
This is an entirely new file even though the same string lives inside. We can check after we merge what's going to trigger the crowdin bot and what are going to be the updates.

Reply:
Solved it by running the Crowdin action after creating the new files, before deleting the old ones. Seems like Crowdin is then able to fill in the duplicate entries properly.